lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -312,6 +274,7 @@ public function get_item_schema() {
 				'id'                   => array(
 					'description' => __( 'Unique identifier for the post.', 'default' ),
 					'type'        => 'integer',
+					'context'     => array( 'edit' ),
```

Comment:
Adding support for the context param. Font families are not embedded, so the only context is `edit`.